metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/powerbi_api.py

```diff
@@ -305,11 +307,42 @@ def _get_workspace_datasets(self, scan_result: Optional[dict]) -> dict:
                             table[Constant.NAME].replace(" ", "_"),
                         ),
                         expression=expression,
+                        columns=self.get_columns(table.get("columns", [])),
+                        measures=self.get_measures(table.get("measures", [])),
                     )
                 )
 
         return dataset_map
 
+    @staticmethod
+    def get_measures(measures: List[dict]) -> List[Measure]:
+        results = []
+        for measure in measures:
+            ms = Measure(
+                name=measure.get(Constant.NAME, ""),
```

Comment:
Make it optional and use None as default value. Make similar changes all other places.